services/hydra/declarative-projects.nix

```diff
@@ -30,5 +30,12 @@
       description = "Packages for Redox";
       homepage = "https://github.com/nix-community/redoxpkgs";
     };
+    rust-for-linux = {
+      displayName = "Rust For Linux";
+      inputValue = "https://github.com/rust-for-linux/nix";
+      specFile = ".hydra/spec.json";
```

Comment:
Note you will need to rename it in your repository (where it is `specs.json`).

Reply:
Oh, thanks :-)

Reply:
Fixed in my repo, thank's for that.